### ISSUE TITLE
Postgres error

### ISSUE THREAD

**Issue by 0ssigeno** [intelowlproject/IntelOwl#218]
When analyzing some observable (i.e. bit.ly), IntelOwl crashes when trying to save the analyzer output, reporting
```
  "unsupported Unicode escape sequence\nLINE 1: UPDATE \"api_app_job\" SET \"analysis_reports\" = '[{\"name\": \"CI...\n                                                      ^\nDETAIL:  \\u0000 cannot be converted to text.\nCONTEXT:  JSON data, line 1: ... America\", \"address\": \"bit.ly\", \"indicator_link\":...\n"
```
Moreover, since the execution crashes halfway through the analysis, the output of each analyzer after the one that causes the crash are not reported/executed.
 
Unfortunely, I have no more example to provide.
![image](https://user-images.githubusercontent.com/30528879/95753201-8d033e00-0ca1-11eb-807e-b4b6fc4af285.png)

**Comment by mlodic** [comment]
Asked here too #138

**Comment by mlodic** [comment]
fixed here #223 and available in the next release
